genesyscloud/util/lists/util_lists.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-func StringInSlice(a string, list []string) bool {
+func ItemInSlice[T comparable](a T, list []T) bool {
```

Comment:
Nice.  Generics are very cool for solving boilerplate code.  

You can even use them in interfaces